assets/js/src/core/modules/element/dynamic-types/definitions/objects/data-related/components/many-to-many-relation/grid.tsx

```diff
@@ -94,7 +95,20 @@ export const ManyToManyRelationGrid = forwardRef(function ManyToManyRelationGrid
             editable: false,
             config: getElementCellConfig(props.inherited === true || props.disabled === true)
           },
-          size: 200
+          size: 200,
+          cell: (info) => {
+            return (
+              <Flex
+                align={ 'center' }
+                className={ 'p-mini' }
+              >
+                <Tag
```

Comment:
If we have a path formatter involved we should not add the blue tag but just directly output the HTML as it is.

![image](https://github.com/user-attachments/assets/d596bbc5-6625-4114-8d23-761ee7f0cd23)